diff --git a/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.cjs.js b/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.cjs.js
index 6185d98..2981682 100644
--- a/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.cjs.js
+++ b/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.cjs.js
@@ -6052,7 +6052,7 @@ function useKeyboardSensor(api) {
 var idle$2 = {
   type: 'IDLE'
 };
-var timeForLongPress = 120;
+var timeForLongPress = 1;
 var forcePressThreshold = 0.15;
 
 function getWindowBindings(_ref) {
diff --git a/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.esm.js b/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.esm.js
index 351b787..d84ce4b 100644
--- a/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.esm.js
+++ b/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.esm.js
@@ -6045,7 +6045,7 @@ function useKeyboardSensor(api) {
 var idle$2 = {
   type: 'IDLE'
 };
-var timeForLongPress = 120;
+var timeForLongPress = 1;
 var forcePressThreshold = 0.15;
 
 function getWindowBindings(_ref) {
diff --git a/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.js b/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.js
index 3061e49..2d44cef 100644
--- a/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.js
+++ b/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.js
@@ -9049,7 +9049,7 @@
   var idle$2 = {
     type: 'IDLE'
   };
-  var timeForLongPress = 120;
+  var timeForLongPress = 1;
   var forcePressThreshold = 0.15;
 
   function getWindowBindings(_ref) {
diff --git a/node_modules/react-beautiful-dnd/src/view/use-sensor-marshal/sensors/use-touch-sensor.js b/node_modules/react-beautiful-dnd/src/view/use-sensor-marshal/sensors/use-touch-sensor.js
index 01f87a9..2e167e9 100644
--- a/node_modules/react-beautiful-dnd/src/view/use-sensor-marshal/sensors/use-touch-sensor.js
+++ b/node_modules/react-beautiful-dnd/src/view/use-sensor-marshal/sensors/use-touch-sensor.js
@@ -45,7 +45,7 @@ type Phase = Idle | Pending | Dragging;
 const idle: Idle = { type: 'IDLE' };
 // Decreased from 150 as a work around for an issue for forcepress on iOS
 // https://github.com/atlassian/react-beautiful-dnd/issues/1401
-export const timeForLongPress: number = 120;
+export const timeForLongPress: number = 1;
 export const forcePressThreshold: number = 0.15;
 
 type GetBindingArgs = {|
